name: Supervised Classification, features: sick
{'X_train': {'cols': 0.013880481571159536, 'rows': 0.2029190207156309}, 'X_valid': {'cols': 0.011164917944578962, 'rows': 0.1807909604519774}, 'X_test': {'cols': 0.02109528859263748, 'rows': 0.28207847295864263}}
dataset sizes
(4310, 28) (1062, 28) (943, 28)
dropped dataset sizes
(3444, 28) (870, 28) (677, 28)
key: 9927, k: 1/4, dataset: 38, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 38, 'max_bin': 250, 'max_depth': 10, 'min_data_in_leaf': 13, 'learning_rate': 0.02796201100840992, 'num_iterations': 324, 'objective': 'softmax', 'verbose': -1, 'num_class': 2}
Training until validation scores don't improve for 100 rounds
Did not meet early stopping. Best iteration is:
[265]	valid_0's multi_logloss: 0.0114869
strategy:None, acc gbm: 0.9883351007423118
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
strategy:None, nll xbg:0.06380549818277359
name: Supervised Classification, features: sick
{'X_train': {'cols': 0.013934619002822201, 'rows': 0.19708372530573848}, 'X_valid': {'cols': 0.01268796992481203, 'rows': 0.20018796992481203}, 'X_test': {'cols': 0.020565065899106196, 'rows': 0.29056203605514314}}
dataset sizes
(4254, 28) (1064, 28) (943, 28)
dropped dataset sizes
(3416, 28) (851, 28) (669, 28)
key: 2475, k: 2/4, dataset: 38, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 38, 'max_bin': 250, 'max_depth': 10, 'min_data_in_leaf': 13, 'learning_rate': 0.02796201100840992, 'num_iterations': 324, 'objective': 'softmax', 'verbose': -1, 'num_class': 2}
Training until validation scores don't improve for 100 rounds
Did not meet early stopping. Best iteration is:
[265]	valid_0's multi_logloss: 0.00331763
strategy:None, acc gbm: 0.9893955461293743
strategy:None, nll xbg:0.07782596349716187
name: Supervised Classification, features: sick
{'X_train': {'cols': 0.013078470824949697, 'rows': 0.18896713615023475}, 'X_valid': {'cols': 0.011659072634682391, 'rows': 0.16885553470919323}, 'X_test': {'cols': 0.02238297227692774, 'rows': 0.3149522799575822}}
dataset sizes
(4268, 28) (1066, 28) (943, 28)
dropped dataset sizes
(3461, 28) (886, 28) (646, 28)
key: 7457, k: 3/4, dataset: 38, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 38, 'max_bin': 250, 'max_depth': 10, 'min_data_in_leaf': 13, 'learning_rate': 0.02796201100840992, 'num_iterations': 324, 'objective': 'softmax', 'verbose': -1, 'num_class': 2}
Training until validation scores don't improve for 100 rounds
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
Early stopping, best iteration is:
[209]	valid_0's multi_logloss: 0.0145954
strategy:None, acc gbm: 0.9840933191940615
strategy:None, nll xbg:0.12790332734584808
name: Supervised Classification, features: sick
{'X_train': {'cols': 0.01357733175914994, 'rows': 0.19834710743801653}, 'X_valid': {'cols': 0.013422366113584244, 'rows': 0.20113314447592068}, 'X_test': {'cols': 0.022761702772307223, 'rows': 0.30965005302226933}}
dataset sizes
(4270, 28) (1059, 28) (943, 28)
dropped dataset sizes
(3425, 28) (846, 28) (651, 28)
key: 8500, k: 4/4, dataset: 38, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 38, 'max_bin': 250, 'max_depth': 10, 'min_data_in_leaf': 13, 'learning_rate': 0.02796201100840992, 'num_iterations': 324, 'objective': 'softmax', 'verbose': -1, 'num_class': 2}
Training until validation scores don't improve for 100 rounds
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
Early stopping, best iteration is:
[219]	valid_0's multi_logloss: 0.00786913
strategy:None, acc gbm: 0.9957582184517497
strategy:None, nll xbg:0.023819975554943085